app/controllers/users_controller.rb

```diff
@@ -60,6 +60,7 @@ def update
   # DELETE /users/1
   def destroy
     user = User.find(params[:id])
+    authorize user
```

Comment:
Curious as to why this was added, was it just missing from before or is it something to do with the new restore feature?